src/ut/cache_test.cpp

```diff
@@ -1897,4 +1897,159 @@ TEST_F(CacheRequestTest, DissociateImplicitRegistrationSetFromWrongImpi)
   EXPECT_TRUE(log.contains("not all the provided IMPIs are associated with the IMPU"));
 }
 
+//
+// Tests for listing IMPUs in the cache.
+//
+
+// Helper class to save off the IMPUs retrieved from the database.
+class ListImpusRecorder : public ResultRecorderInterface
+{
+public:
+  void save(CassandraStore::Operation* op)
+  {
+    impus = dynamic_cast<Cache::ListImpus*>(op)->get_impus_ref();
+  }
+  std::vector<std::string> impus;
+};
+
```

Comment:
Double blank line !!!! I'd move one of these blank lines up above `std::vector<std::string> impus;` FWIW.